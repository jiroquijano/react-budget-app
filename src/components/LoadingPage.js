import React from 'react';

const LoadingPage = ()=>(
    <div className="loader">
        <img classname="loader__image" src={"/images/loader.gif"}/>
    </div>
);

export default LoadingPage;