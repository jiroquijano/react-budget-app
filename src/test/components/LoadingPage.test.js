import {shallow} from 'enzyme';
import React from 'react';
import LoadingPage from '../../components/LoadingPage';

test ("LoadingPage should render properly",()=>{
    const wrapper = shallow(<LoadingPage/>);
    expect(wrapper).toMatchSnapshot();
});