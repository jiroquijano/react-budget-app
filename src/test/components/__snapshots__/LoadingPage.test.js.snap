// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should render properly 1`] = `
<div
  className="loader"
>
  <img
    classname="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
